frontend/src/metabase/timelines/components/EventList/EventList.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useMemo } from "react";
+import _ from "underscore";
+import { t } from "ttag";
+import { parseTimestamp } from "metabase/lib/time";
+import { Collection, Timeline } from "metabase-types/api";
+import EventCard from "../EventCard";
+import {
+  ListFooter,
+  ListIcon,
+  ListIconContainer,
+  ListIconText,
+  ListThread,
+  ListThreadContainer,
+} from "./EventList.styled";
+
+export interface EventListProps {
+  timeline: Timeline;
+  collection: Collection;
+}
+
+const EventList = ({ timeline, collection }: EventListProps): JSX.Element => {
+  const allEvents = timeline.events;
+  const sortedEvents = useMemo(
```

Comment:
I'm going to revise this solution in a follow-up PR with archiving. Basically there will be `events` prop and sorting will be done outside of the component.